msg/offboard_control_mode.msg

```diff
@@ -5,6 +5,9 @@ uint64 timestamp		# time since system start (microseconds)
 bool ignore_thrust
 bool ignore_attitude
 bool ignore_bodyrate
```

Comment:
Remove? Should be unused now.

Reply:
you mean ignore_bodyrate? Yes can check on that and remove it

Reply:
Fixed. Removed it and build checks out.